examples/nextjs-spa/src/pages/recovery.tsx

```diff
@@ -84,10 +84,7 @@ const Recovery: NextPageWithLayout = () => {
         updateRecoveryFlowBody: values,
       })
       .then(({ data }) => {
-        // reset the form data completely
-        setFlow(null)
-        // Form submission was successful, show the message to the user!
-        getFlow(data.id)
```

Comment:
Reverting all of these changes from https://github.com/ory/elements/commit/85cc97964328ff44488d10b35e385c070f2b2f5f because they just masked the real issue.